index.bs

```diff
@@ -6789,7 +6798,8 @@ may return any value, which will be discarded.
     1.  Otherwise:
         1.  Let |lowerBound| be -2<sup>|bitLength| − 1</sup>.
         1.  Let |upperBound| be 2<sup>|bitLength| − 1</sup> − 1.
-    1.  Set |x| to [=?=] [=ToNumber=](|V|).
+    1.  Let |x| be [=?=] [=ToNumber=](|V|).
+    1.  If |x| is −0 or |x| is +0, then set |x| to 0.
```

Comment:
I was unsure whether +0 was considered the same as 0, or not.

Was expecting for this to be raised here.